core/modules/file/file.theme.inc

```diff
@@ -278,6 +279,14 @@ function theme_file_upload_help($variables) {
   if (isset($upload_validators['file_validate_extensions'])) {
     $descriptions[] = t('Allowed file types: !extensions.', array('!extensions' => '<strong>' . check_plain($upload_validators['file_validate_extensions'][0]) . '</strong>'));
   }
+  if (isset($cardinality)) {
```

Comment:
Can we please move this block of text above the file size and file types/extensions lines?